C2GUILauncher/src/SigFileHelper.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using log4net;
+
+namespace C2GUILauncher.Mods
+{
+    public static class SigFileHelper
+    {
+        private static readonly ILog log = LogManager.GetLogger(typeof(SigFileHelper));
+
+        private static string PakDirectory => FilePaths.PakDir;
+        private static string DefaultSigFile => Path.Combine(PakDirectory, "pakchunk0-WindowsNoEditor.sig");
+
+        private static Dictionary<string, (bool HasPak, bool HasSig)> GetPakAndSigFiles()

```

Comment:
This type should be `Dictionary<string, (bool HasPak, bool HasSig)>?` to indicate that it may return null